src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,29 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {post.title}
+      </h3>
+
+      <p>
+        {' Posted by  '}
+      </p>
+
+      <UserInfo user={post.user} />
```

Comment:
```suggestion
      {user && <UserInfo user={post.user} />}
```
potentially user can be undifined. so better to render UserInfo only when user exists.